extensions/amp-iframe/1.0/README.md

```diff
@@ -0,0 +1,197 @@
+# Bento Iframe
+
+## Usage
+
+Displays an iframe.
+
+Use Bento Iframe as a web component `<bento-iframe>`, or as a Preact/React functional component `<BentoIframe>`.
+
+### Web Component
+
+You must include each Bento component's required CSS library to guarantee proper loading and before adding custom styles. Or use the light-weight pre-upgrade styles available inline. See [Layout and style](#layout-and-style).
+
+The examples below demonstrate use of the `<bento-iframe>` web component.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-iframe
+```
+
+```javascript
+import '@ampproject/bento-iframe';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <script async custom-element="bento-iframe" src="https://cdn.ampproject.org/v0/bento-iframe-1.0.js"></script>
+  <link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-iframe-1.0.css">
+  <style data-bento-boilerplate>
+    bento-iframe {
+      display: block;
+      overflow: hidden;
+      position: relative;
+    }
+  </style>
+</head>
+<bento-iframe src="https://en.wikipedia.org/wiki/Bento" width="800" height="600">
+</bento-iframe>
+
+<script>
+  (async () => {
+    const fitText = document.querySelector('#my-fit-text');
+    await customElements.whenDefined('bento-iframe');
+  })();
```

Comment:
Please also add a mutability example, i.e. where you `setAttribute('src', ...)` to a new url on button click, which should demonstrate that the iframe will reload with the new src.

Reply:
Done